android/shared/src/commonMain/kotlin/io/rebble/cobble/shared/ui/view/home/watches/WatchesPage.kt

```diff
@@ -0,0 +1,100 @@
+
+    if (selectedWatch != null) {
+
+        WatchBottomSheetContent(
+                watch = selectedWatch,
+                onToggleConnection = { viewModel.toggleConnection(selectedWatch, true) },
+                onForgetWatch = { viewModel.forgetWatch(selectedWatch) },
+                onCheckForUpdates = { viewModel.checkForUpdates(selectedWatch) },
+                clearSelection = { viewModel.clearSelection() }
+        )
+    }
+
+    Column(
+            modifier = Modifier.fillMaxSize(),
+            verticalArrangement = Arrangement.Top
+    ) {
+        CenterAlignedTopAppBar(
+                colors = TopAppBarDefaults.centerAlignedTopAppBarColors(
+                        containerColor = MaterialTheme.colorScheme.primaryContainer,
+                ),
+                title = {
+                    Text(
+                            "My watches",
+                            maxLines = 1,
+                            overflow = TextOverflow.Ellipsis
+                    )
+                }
+        )
+        if (viewModel.getConnectedWatch() == null) {
+            Row(
+                    modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(horizontal = 16.dp, vertical = 24.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.spacedBy(25.dp)
+            ) {
+                Box(
+                        modifier = Modifier
+                                .size(60.dp)
+                                .background(
+                                        color = MaterialTheme.colorScheme.primaryContainer,
+                                        shape = RoundedCornerShape(8.dp)
+                                ),
+                        contentAlignment = Alignment.Center
+                ) {
+                    RebbleIcons.disconnectFromWatch()
+                }
+
+                Column {
+                    Text(
+                            text = "Nothing connected",
+                            style = MaterialTheme.typography.bodyLarge,
+                            fontWeight = FontWeight.SemiBold
+                    )
+                    Text(
+                            text = "Background service stopped",
+                            style = MaterialTheme.typography.bodyMedium,
+                            color = MaterialTheme.colorScheme.onSurfaceVariant
+                    )
+                }
+            }
+        } else {
+            WatchesListItem(watch = viewModel.getConnectedWatch()!!, viewModel = viewModel)
+        }
+        Text(modifier = Modifier
+                        .padding(
+                        horizontal = 10.dp),
+                        text = "Other Watches")
+        HorizontalDivider(thickness = 2.dp, color = MaterialTheme.colorScheme.secondary)
+
+        LazyColumn {
+            items(viewModel.watches.filter { !it.isConnected }) { watch ->
```

Comment:
It is a very good idea to define a key for each item in LazyColumn 
https://developer.android.com/develop/ui/compose/lists#item-keys